app/rpc/namespaces/eth/api.go

```
@@ -506,13 +513,42 @@ func (api *PublicEthereumAPI) SendRawTransaction(data hexutil.Bytes) (common.Has
 		return common.Hash{}, nil
 	}
 
+	// send chanData to txPool
+	{
+		// Get sender address
+		chainIDEpoch, err := ethermint.ParseChainID(api.clientCtx.ChainID)
+		if err != nil {
+			return common.Hash{}, nil
```

should return the error encountered